fb/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    if @comment.save
+      redirect_back(fallback_location: root_path)
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @comment = Comment.find_by_id(params[:format])
+    if current_user.id == @comment.user_id
```

Comment:
You should check if the `@comment` even exists first [OPTIONAL]

Reply:
Sure, I will handle this edge case.